src/components/investment-cards/UniswapPoolListRow.js

```diff
@@ -26,12 +26,7 @@ const TopRowContainer = styled(Row).attrs({
   justify: 'flex-start',
 })``;
 
-const PriceContainer = ios
-  ? View
-  : styled(View)`
-      margin-top: -3;
-      margin-bottom: 3;
-    `;
+const PriceContainer = View;
```

Comment:
@estebanmino instead of doing this, i would prefer if you used the `<View />` directly on Line 47 (https://github.com/rainbow-me/rainbow/blob/df3a9d83ee174beccc29980db07ed7d9a79c832f/src/components/investment-cards/UniswapPoolListRow.js#L47)

Reply:
@mikedemarais makes sense and done 👌